src/console.c

```diff
@@ -0,0 +1,17 @@
+#include "system.h"
+
+#include "console.h"
+
+bool use_ansi_escapes(void) {
+#ifndef HAVE_WINDOWS
+	const char *term = getenv("TERM");
+	return term && strcmp(term, "dumb") && isatty(STDERR_FILENO);
```

Comment:
If you run the `tinc log` command it's going to output the log to `stdout`, so then you should check `STDOUT_FILENO` instead. Maybe pass the desired `FILE *` (`stdout`/`stderr`) to this function as a parameter?